test/e2e/network/networking.go

```diff
@@ -52,6 +52,7 @@ func checkConnectivityToHost(f *framework.Framework, nodeName, podName, host str
 
 	pod := e2epod.NewAgnhostPod(f.Namespace.Name, podName, nil, nil, nil)
 	pod.Spec.Containers[0].Command = command
+	pod.Spec.Containers[0].Args = nil // otherwise 'pause` is magically an argument to nc, which causes all hell to break loose
```

Comment:
Argv is usually at least `[]string{"binary"}`, I think we want to instead change the behavior of:
https://github.com/kubernetes/kubernetes/blob/4d757f1c527b28c3cd963329e4add239ddcfb60c/test/e2e/framework/pod/resource.go#L435-L438

Or pass an explicit `[]string{"nc"}`.

In fact, it seems like the correct layering here is to do `s/command/args/` in general, the command here should likely be passed as the args (docker CMD) instead of command (ENTRYPOINT), leaving room for entrypoint hijinks later.

Reply:
i just tried to urbandictionary `WAI` and couldnt figure out what it is :) ?